test/model.test.js

```diff
@@ -0,0 +1,11 @@
+const itemsCreationModel = require('../src/model')
+const assert = require('assert').strict;
+
+describe('integration test', function() 
+{
+  it("should be able to add and complete TODOs", function() 
+  {
+    let todos = new itemsCreationModel();
```

Comment:
Just FYI: itemsCreationModel is a so called factory function. That means you invoke like this: `let todos = itemsCreationModel()`. You use the new keyword if you want to create instances from constructor functions and classes.

Reply:
Thanks for clarification ;-)